daemon/daemon.go

```diff
@@ -760,9 +760,23 @@ func (daemon *Daemon) Shutdown() error {
 
 				go func() {
 					defer group.Done()
-					// If container failed to exit in 10 seconds of SIGTERM, then using the force
-					if err := c.Stop(10); err != nil {
-						logrus.Errorf("Stop container %s with error: %v", c.ID, err)
+					if c.IsPaused() {
```

Comment:
@jhowardmsft what would be the best way of making this platform agnostic?
`syscall.SIGTERM` exists on windows, but is an *invented* value. should we just put this whole addition in a method that's empty if the file has the windows buildtag?